nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -1313,12 +1313,30 @@ impl CompileEvaluation for ast::StaticMemberExpression<'_> {
             ctx.exe.add_instruction(Instruction::GetValue);
         }
 
+        ctx.exe.add_instruction(Instruction::LoadCopy);
+        if self.optional {
+            ctx.exe.add_instruction(Instruction::IsNullOrUndefined);
+        }
+
+        let jump_to_access = ctx
+            .exe
+            .add_instruction_with_jump_slot(Instruction::JumpIfNot);
+
+        ctx.exe.add_instruction(Instruction::Store);
         // 4. Return EvaluatePropertyAccessWithIdentifierKey(baseValue, IdentifierName, strict).
         let identifier = String::from_str(ctx.agent, self.property.name.as_str());
         ctx.exe.add_instruction_with_identifier(
             Instruction::EvaluatePropertyAccessWithIdentifierKey,
             identifier,
         );
+
+        let jump_to_end = ctx.exe.add_instruction_with_jump_slot(Instruction::Jump);
+        ctx.exe.set_jump_target_here(jump_to_access);
+
```

Comment:
**issue:** I think here you need to pop the value from the stack that LoadCopy put there.